CONTRIBUTING.md

```diff
@@ -1,25 +1,49 @@
+# Guidelines for Contributing to CraftOS-Standards
 
-## Guidelines for Contributing to CraftOS-Standards
-
-Welcome, fellow developer! First off, make yourself familiar with the [tutorial](./Tutorial/GettingStarted.md). It's a long read, so enjoy it with some :coffee:. If you'd like to create new standards, the tutorial should actually give you all that you'll need to know. Thanks for helping us out by standardizing something awesome!
+Welcome, fellow developer! First off, make yourself familiar with the [tutorial](./Tutorial/GettingStarted.md). It's a
+long read, so enjoy it with some :coffee:. If you'd like to create new standards, the tutorial should actually give you
+all that you'll need to know. Thanks for helping us out by standardizing something awesome!
```

Comment:
I'll fix this but honestly this entire thing needs a rewrite and I don't think this is the appropriate PR for this.